web/src/tippyjs.ts

```diff
@@ -47,6 +48,22 @@ export const LONG_HOVER_DELAY: [number, number] = [750, 20];
 // keyboard shortcut. For these tooltips, it's very important to avoid
 // distracting users unnecessarily.
 export const EXTRA_LONG_HOVER_DELAY: [number, number] = [1500, 20];
+export const topic_visibility_tooltip_prop = {
+    delay: LONG_HOVER_DELAY,
+    appendTo: () => document.body,
+    onShow(instance: tippy.Instance) {
+        const $elem = $(instance.reference);
+        const current_visibility_policy_str = $elem.attr("data-tippy-content");
+        instance.setContent(
+            ui_util.parse_html(
+                render_change_visibility_policy_button_tooltip({current_visibility_policy_str}),
+            ),
+        );
+    },
+    onHidden(instance: tippy.Instance) {
+        instance.destroy();
+    },
+};
```

Comment:
Why is this here? It definitely shouldn't have this prime visual real estate at the top of the file.

Reply:
Probably reasonable to just put it just after `tippy.default`. 

Please also call it `topic_visibility_tooltip_props`; it shouldn't be singular since it's a collection.